### PR TITLE
Add CORSID

### PR DESCRIPTION
**Pull request by ChuanyiZ**
Add CORSID

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by ChuanyiZ** [comment]
@BiocondaBot please add label

File: recipes/corsid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9330e919c3384d1bcf86d0964d826d16f1fc3b68325b50d48e5e2bb4e1ae371f
+
+build:
+  noarch: generic
```
Comment (dpryan79) [inline recipes/corsid/meta.yaml:13]: This should be `noarch: python`

File: recipes/corsid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  imports:
+    - corsid
+
+about:
+  home: "http://github.com/elkebir-group/CORSID"
+  license: MIT
+  license_family: MIT
+  summary: "Core Sequence Identifier"
+  doc_url:
```
Comment (dpryan79) [inline recipes/corsid/meta.yaml:41]: Please remove the doc_url line, since it's empty

**Review by dpryan79 — CHANGES_REQUESTED** [review]
A couple minor changes, please see comments.

**Comment by ChuanyiZ** [comment]
@BiocondaBot please add label

File: recipes/corsid/meta.yaml (hunk 2 above)
Comment (ChuanyiZ) [reply recipes/corsid/meta.yaml]: Fixed, thank you

File: recipes/corsid/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - python
+    - setuptools
+  host:
+    - numpy
+    - pandas
+    - pip
+    - pysam
+    - pytablewriter
+    - python
+    - tqdm
```
Comment (simonbray) [inline recipes/corsid/meta.yaml:28]:
```suggestion
  host:
    - pip
    - python
```
a build section is not needed for python packages and only python and pip are needed in host

Comment (ChuanyiZ) [reply recipes/corsid/meta.yaml]: Thank you

**Review by simonbray — APPROVED** [review]

**Comment by simonbray** [comment]
@dpryan79 I think you need to approve

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge
Thanks for the ping @simonbray !

**Pull request merged**